### PR TITLE
Avoid silent conversion of service_port in ingress trait & replicaCount in manualscaler trait

### PR DESCRIPTION
**Pull request by iyyappam**
Issue: #477

If the values are provided as string, try converting them,
else add warning to the log and set it to default value.

This may not be needed if we add the validate the config schema using validation schema which is in the works.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -52,6 +52,7 @@ fn main() -> Result<(), Error> {
 
     let top_ns = std::env::var("KUBERNETES_NAMESPACE").unwrap_or_else(|_| DEFAULT_NAMESPACE.into());
     let top_cfg = kubeconfig().expect("Load default kubeconfig");
+    info!("apiserver:{}", top_cfg.base_path);
```
Comment (wonderflow) [inline src/main.rs]: is this a debug message?

Comment (iyyappam) [reply src/main.rs]: I intentionally made it as info as we want to know the apiserver all the time.

Comment (wonderflow) [reply src/main.rs]: it's ok

**Comment by iyyappam** [comment]
@wonderflow , just checking to see if you are waiting for something from me before the merge.

**Comment by wonderflow** [comment]
@iyyappam  I just wait if someone else want to review. It's OK for me to merge it.

**Pull request merged**